src/core/backend_context.cc

```diff
@@ -165,34 +165,26 @@ BackendContext::SetFixedSizeOutputBuffer(
     if (payload.status_.IsOk() && (payload.response_provider_ != nullptr) &&
         payload.response_provider_->RequiresOutput(name)) {
       auto dst_memory_type = src_memory_type;
+      int64_t dst_device_id;
       void* buffer = nullptr;
 
       // try to get buffer with the same memory type as the output tensor
       Status status = payload.response_provider_->AllocateOutputBuffer(
           name, &buffer, expected_byte_size, content_shape, src_memory_type,
-          src_memory_type_id);
-
+          src_memory_type_id, &dst_memory_type, &dst_device_id);
+      LOG_VERBOSE(1) << "src_memory_type: " << src_memory_type;
```

Comment:
Make a single, more expressive log message or remove.

Reply:
Used this for logging. Removed